game.py

```diff
@@ -14,13 +14,67 @@
 
 
 def snowman(snowman_word):
-    """Complete the snowman function
-    replace "pass" below with your own code
-    It should print 'Congratulations, you win!'
-    If the player wins and, 
-    'Sorry, you lose! The word was {snowman_word}' if the player loses
-    """
-    pass
+
+    # print(snowman_word)
```

Comment:
Remove code that is not used and commented out. Leaving commented out code can lead to confusion when other developers come along and read code. Why is this commented out? Should I comment it back in?

Or worse, if the code gets accidentally commented in then you would have code that shouldn't execute that ends up getting run which would lead to a bug. In this case, it's a minor bug that prints a string, but if the code was something more consequential like deleting a user account then we'd be dealing with a bigger issue.

```suggestion
```